### ISSUE TITLE
El Capitan

### ISSUE THREAD

**Issue by Sydney-o9** [lsd/iMacMonitor#2]
I'm having trouble making it work with El Capitan... has anyone managed to use it in El Capitan?
Thank you!

**Comment by elrrrrrrr** [comment]
+1

**Comment by lsd** [comment]
I don't have El Capitan at the moment but will take a look as soon as I can.

**Comment by elrrrrrrr** [comment]
It seems to be working . 
OS version 10.11.1(15B42)

3x

**Comment by lsd** [comment]
@elrrrrrrr  Thanks for taking a look and posting back. @Sydney-o9 did you run into any specific issue getting this to work?

If no workaround is needed I'll close this and update the readme to say it's compatible with  10.11 and other versions I'll test it on.

**Comment by Sydney-o9** [comment]
Yes my screen was totally black after rebooting. Feel free to close if you guys manage to make it work. It might just be me not going through the steps correctly. Thank you for your help anyway!

**Comment by rjgrazioli** [comment]
Doens't seem to work for me either. El Captian 2011 iMac to El Capitan 2015 MacBook Pro 13" Retina—just stays asleep.
